### ISSUE TITLE
Unable to achieve the expected result when training

### ISSUE THREAD

**Issue by wytcsuch** [Alpha-VLLM/Lumina-Image-2.0#49]
Thank you very much for your outstanding contribution.I am training a 256 pixel model from scratch.In order to improve training efficiency, I reduced the size of the model：
```
NextDiT(
        patch_size=2,
        dim=1152,
        n_layers=28,
        n_heads=16,
        n_kv_heads=8,
        axes_dims=[24, 24, 24],
        axes_lens=[350, 512, 512],
        **kwargs
    )
```
I use clip+t5 as text embedding, totally 308 tokens. I keep training hyperparameters consistent with yours.The training data is 34w, and a total of 100 epochs were trained, the loss curve is as follows:
![Image](https://github.com/user-attachments/assets/f18cd65b-a5d3-4abb-8cdb-3eb06e4b1c66)
Serious loss of image details：
![Image](https://github.com/user-attachments/assets/e57a5f78-d185-49d6-894f-db955cb08dde)
![Image](https://github.com/user-attachments/assets/42ecb8a1-0ac3-46f1-8b4c-3718da32bafb)
![Image](https://github.com/user-attachments/assets/0c2db214-4ef1-4ae5-b5a7-94291709c239)
Here is inference hyperparameters:
```
from transport import Sampler, create_transport
transport = create_transport(
        "Linear",
        "velocity",
        None,
        None,
        None,
    ) 
sampler = Sampler(transport)

sample_fn = sampler.sample_ode(
    sampling_method="midpoint",
    num_steps=16,
    atol=1e-6,
    rtol=1e-3,
    reverse=False,
    time_shifting_factor=6
)
```
